src/main/scala/com/cognite/spark/datasource/DefaultSource.scala

```diff
@@ -191,6 +191,9 @@ class DefaultSource
         new EventsRelation(config)(sqlContext) //.EventsInsertion(config)
       case "timeseries" =>
         new TimeSeriesRelation(config)(sqlContext)
+      case "assets" =>
+        val assetsPath = parameters.get("assetsPath")
```

Comment:
Is this parameter documented?

Reply:
Looks like an older parameter that's no longer in use, let's remove this from AssetsRelation.